images/build/benv.sh

```diff
@@ -79,19 +79,38 @@ benv-versions() {
   done
 }
 
+benv-getVersions() {
+    local rootDir="$1"
+    local versions=""
+    for i in $(find "$rootDir" -maxdepth 1)
+    do
+        local name=$(basename "$i")
+        if [ ! -z "$versions" ]; then
+            versions+=","
+        fi
+        versions+="$name"
+    done
+    echo $versions
+}
+
 benv-resolve() {
-  local name=$(echo $1 | sed 's/=.*$//')
-  local value=$(echo $1 | sed 's/^.*=//')
+  # Splits the string based on the first occurrence of '='
+  # First occurrence is important because the value could have a '=' itself
+  # For example, benv dotnet="=2.1.14"
+  IFS="=" read -r name value <<< "$1"
 
   # Resolve node versions
   if matchesName "node" "$name" || matchesName "node_version" "$name" && [ "${value::1}" != "/" ]; then
-    if [ ! -d "/opt/nodejs/$value" ]; then
+    versions=`benv-getVersions /opt/nodejs`
+    resolvedVersion=`oryx resolveVersion "$value" "$versions"`
```

Comment:
we're adding a circular dependency back to oryx command. Can we simplify this design so that benv doesn't depend on oryx?